core/src/services/redis/backend.rs

```diff
@@ -439,10 +439,10 @@ impl kv::Adapter for Adapter {
         let conn = self.conn().await?;
         match conn {
             RedisConnection::Normal(mut conn) => {
-                conn.append(key, value).await.map_err(format_redis_error)?;
+                () = conn.append(key, value).await.map_err(format_redis_error)?;
```

Comment:
Could you share the details of the lint issue? I think it might be a false positive.

Reply:
See commit message

Reply:
Specifically, `conn.append` returns `T: FromRedisValue`. The statement `conn.append();`'s type inference will lead to `!`. The change makes it inference to `()`.

Reply:
Thanks, this line of code is quite weird since `()` should normally be safe to ignore.